### PR TITLE
Remove overwriting of DOI and PMID with values found in OpenAlex in _ids files

### LINKED ISSUE #101: Find OpenAlexID with incorrect PMID and correct DOI
Currently, when both the PMID and the DOI are available in a dataset, the enrich.py script will first try to find the OpenAlexID based on PMID. 
It will overwrite the current PMID and DOI of the records with a PMID, even if no OpenAlexID (and thus no DOI) was found. 
This results in an empty DOI, which will no longer be searched on.

Given that in all following steps we only use the OpenAlexID, I suggest we simply do not overwrite the PMID and DOI we retrieve from OpenAlex. This solves the problem and has the benefit that we maintain the ID that was present in the original data and was searched on.

### LINKED ISSUE #110: Method not always stored in the enrich.py
Enrich.py adds a column method with the identifier used.
Currently however this is only filled for title (+year), not for doi or pmid.

### PR DESCRIPTION
**Pull request by EmilyWes**
Solves issues 101 and 110. (I am not allowed to link them via the development, probably since the PR is not to the main branch)
Closes #101
Closes #110

### CODE REVIEW

**Comment by J535D165** [comment]
I like the proposal for #110. 

The proposal for #101 will have a side effect, which I'm still determining. In the current situation, the DOIs and PMIDs are always identical to the OpenAlex data after enrichment. In this proposal, this will no longer be the case (which can be confusing as for title search it will overwrite). It might be better to skip the update of the data when there is no OAID for a PMID.

**Comment by EmilyWes** [comment]
I agree, will change the solution.

**Comment by EmilyWes** [comment]
I updated the code so it does not overwrite for the None values.
It really feels like there should be a better way, but I did not really find it...
Any ideas that look better than this?
Or should we maybe change the current setup of openalex_work_by_id?

**Comment by EmilyWes** [comment]
Changed the approach to keep the user input data rather than the pmid/doi provided by OpenAlex. We still retrieve those values later.
This simplifies the code.

File: scripts/enrich.py [hunk 1]
```diff
@@ -168,37 +164,15 @@ def openalex_work_by_id(
...
-        store.append((doi, pmid, oaid))
-
-    return (
-        list(map(lambda x: x[0], store)),
-        list(map(lambda x: x[1], store)),
-        list(map(lambda x: x[2], store)),
-    )
+        store.append(oaid)
 
+    return (list(store))
```
Comment (J535D165) [inline scripts/enrich.py:174]:
```suggestion
    return list(store)
```

**Comment by J535D165** [comment]
For the reader: After merging this PR, the enrich script will no longer modify DOIs and PMIDs in the `_ids` files and will only be at an OA Work ID.

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
🎉

**Pull request merged**